testsuite/MDAnalysisTests/coordinates/test_lammps.py

```diff
@@ -768,6 +768,102 @@ def test_warning(self, system, request):
         with pytest.warns(match="Some of the additional"):
             request.getfixturevalue(system)
 
+    def test_dump_reader_units_attribute(self):
+        """Test that DumpReader has proper units defined"""
+        from MDAnalysis.coordinates.LAMMPS import DumpReader
+
+        expected_units = {
+            "time": "fs",
+            "length": "Angstrom",
+            "velocity": "Angstrom/fs",
+            "force": "kcal/(mol*Angstrom)",
+        }
+
+        assert DumpReader.units == expected_units
+
+    def test_force_unit_conversion_factor(self):
+        """Test that the force conversion factor is correct"""
+        from MDAnalysis import units
+
+        # Get conversion factor from kcal/(mol*Angstrom) to kJ/(mol*Angstrom)
+        factor = units.get_conversion_factor(
+            "force",
+            "kcal/(mol*Angstrom)",  # from (LAMMPS native)
+            "kJ/(mol*Angstrom)",  # to (MDAnalysis base)
+        )
+
+        expected_factor = 4.184  # 1 kcal = 4.184 kJ
+        assert_allclose(factor, expected_factor, rtol=1e-6)
+
+    def test_force_conversion_with_image_vf_file(self):
+        """Test force unit conversion using existing test file with forces"""
+        # Test with convert_units=True (default)
+        u_converted = mda.Universe(
+            LAMMPS_image_vf, LAMMPSDUMP_image_vf, format="LAMMPSDUMP"
+        )
+
+        # Test with convert_units=False
+        u_native = mda.Universe(
+            LAMMPS_image_vf,
+            LAMMPSDUMP_image_vf,
+            format="LAMMPSDUMP",
+            convert_units=False,
+        )
+
+        # Both should have forces
+        assert hasattr(u_converted.atoms, "forces")
+        assert hasattr(u_native.atoms, "forces")
+
+        # Go to last frame where we know forces exist
+        u_converted.trajectory[-1]
+        u_native.trajectory[-1]
+
+        forces_converted = u_converted.atoms.forces
+        forces_native = u_native.atoms.forces
+
+        # Check that forces are different (converted vs native)
+        # The conversion factor should be 4.184
+        expected_factor = 4.184
+        forces_expected = forces_native * expected_factor
+
+        # Test that converted forces match expected values
+        assert_allclose(forces_converted, forces_expected, rtol=1e-6)
+
+        # Test that native forces are unchanged when convert_units=False
+        # Just check they are reasonable values (not zero everywhere)
+        assert not np.allclose(forces_native, 0.0)
```

Comment:
Hmm, this check is perhaps a bit less convincing. What about checking the first and last force with `LAMMPS` or another reader library and ensuring those are preserved in MDA with `convert_units=False`?

Reply:
This is assuming that the forces in native units are zero? Consider using `assert_allclose` with the forces values copied directly from the dump file